.buildkite/scripts/common.sh

```diff
@@ -72,10 +71,6 @@ xpack_packetbeat_changeset=(
   "^x-pack/packetbeat/.*"
   )
 
-xpack_winlogbeat_changeset=(
```

Comment:
That was `xpack/winlogonbeat`. `winlogbeat_changeset` should be removed instead, right?